common/config/azure-pipelines/templates/integration-test-steps.yaml

```diff
@@ -84,7 +84,7 @@ steps:
     displayName: "Run Electron Frontend Tests"
     condition: succeededOrFailed()
 
-  - script: npm run test:integration:backend
+  - script: xvfb-run --auto-servernum --server-args='-screen 0, 1600x900x24' npm run test:integration:backend
```

Comment:
What was the reason for this change?

Reply:
Without display Electron fails to open main window. Not sure why it worked with older Electron versions.